governance/program/src/processor/process_sign_off_proposal.rs

```diff
@@ -29,10 +29,7 @@ pub fn process_sign_off_proposal(program_id: &Pubkey, accounts: &[AccountInfo])
 
     assert_is_valid_realm(program_id, realm_info)?;
 
-    // Governance account data is no longer used in the current version but we still have to load it to validate Realm -> Governance -> Proposal relationship
-    // It could be replaced with PDA check but the account is going to be needed in future versions once we support mandatory signatories
-    // and hence keeping it as it is
-    let _governance_data =
+    let governance_data =
```

Comment:
If there are `governance_data.required_signatories >  0` then assert `proposal.signatories_count` equals to the same number to ensure all required signatories were added to the Proposal before the signing off process starts. Once the first signatory sings off the Proposal enters `SingingOff` state and you can't add more signatories any longer

Reply:
resolved